source/sdk/lang/types.ooc

```diff
@@ -15,10 +15,8 @@ Object: abstract class {
     __destroy__: func {}
 
     free: virtual func {
-      version(!gc) {
-        this __destroy__()
-        gc_free(this)
-      }
+      this __destroy__()
```

Comment:
Soon, this should be removed too, because we shouldn't have any `__destroy__` functions anymore.